config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  root "users#index"
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/posts/:id', to: 'posts#show', as: 'post'
```

Comment:
- [ ] The post show route should be wrapped within the user route. So instead of

  ```
  /posts/:id
  ```
  We should have
  ```
  /users/:user_id/posts/:id
  ```

- **[OPTIONAL]** These routes can be created more easily with `resources`
  
  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```